tests/jest/pageScanner/context.test.js

```
@@ -0,0 +1,63 @@
+			<div id="query-monitor-main">
+				<button id="query-monitor-button"></button>
+			</div>
+			<div class="edac-panel-container">
+				<button id="edac-panel-button"></button>
+			</div>
+		`;
+
+		// Define context with exclude list matching src/pageScanner/index.js
+		const context = {
+			exclude: [ '#wpadminbar', '.edac-panel-container', '#query-monitor-main', '#qm-icon-container' ]
+		};
```

ESLint

[error] Missing trailing comma.

(comma-dangle)